### ISSUE TITLE
Readonly property

### ISSUE THREAD

**Issue by marcalbet** [weareoutman/clockpicker#25]
You could add a readonly property as a clockpicker option like this;:

Options
Name    Default Description
readonly    false         (true, flase)

and then the plugin add automatically this property in input tag:
<input type="text" class="form-control" readonly value="18:00">

**Comment by weareoutman** [comment]
Why not just set this property of the input tag in html manually ?

**Comment by marcalbet** [comment]
You are right but it could be interesting for your great plugin to add this extra option...
